test-data/heat_equation/README.md

```diff
@@ -0,0 +1,6 @@
+# Estimate heat transfer coefficients of a plate using the heat equation as forward model
+
+Based on the following tutorial: https://github.com/equinor/dass/blob/main/notebooks/ES_2D_Heat_Equation.ipynb
+
+- **generate_files.py:** File run once to genereate `CASE.EGRID` and observations.
```

Comment:
I don't fully understand this comment. Maybe like "Please / Do run once ...."

Reply:
No need to run this file unless you want to re-generate the observation files.